src/components/ProjectIssues.js

```diff
@@ -64,11 +65,13 @@ const ProjectIssues = (props) => {
         })
     }
 
+
     if (loadingProjectIssues) return <LoadingProgress/>
     if (errorProjectIssues) {
         return (
             <GithubAccessBlocked
-                message={`You must be a Github collaborator to access this metrics`}
+                message={`You must be a Github collaborator to access this metrics.`}
```

Comment:
Change to `You must have access to this repository on GitHub to access these metrics`